src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayCost = 40;
+  const treeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
+  let totalCost = 0;
+
+  if (days < 3) {
+    totalCost = oneDayCost * days;
```

Comment:
```suggestion
    totalCost = oneDayCost * days;  
    ти можеш винести цю змінну також, так як ми робили з функціями в CSS і тоді тобі не треба ще один цикл для днів до 3х
```

Reply:
Agree, don't repeat yourself: create variable for oneDayCost * days value